Platform/ApollolakeBoardPkg/Script/StitchLoader.py

```diff
@@ -426,7 +426,9 @@ def patch_flash_map (image_data, platform_data = 0xffffffff):
 
             comp  = IFWI_PARSER.locate_component (ifwi, path)
             if not comp:
-                raise Exception("Cannot locate component '%s' in BPDT !" % path)
+                if desc.sig not in [b'_BPM', b'OEMK']:
```

Comment:
Since current SBL code still use _BPM,  should we leave it ?
Also it there a real case that no BPM exists base IFWI ?

Reply:
BPM is used only for BtG purposes.
And, I believe the existing build infrastructure always creates BPM and therefore we see it in all IFWI's.

In the BtG build scenario, this part of the script will be executed after KM and BPM is already built in IFWI and therefore 'BPM' and 'KM' should be there in IFWI image.

Reply:
If BPM will always be there,   "if desc.sig not in [b'OEMK']  should be good enough, is it right ?

Reply:
Agree. I think checking OEMK only would be enough.